.github/workflows/release.yml

```diff
@@ -29,12 +29,14 @@ jobs:
       - name: Create local repository directory
         id: local_repo
         run: echo ::set-output name=dir::"$(mktemp -d repo.XXXXXXXX)"
-      - name: Download Artifacts
-        uses: SonarSource/gh-action_release/download-build@v4
-        with:
-          build-number: ${{ github.event.inputs.buildNumber }}
-          remote-repo: 'sonarsource-helm'
-          local-repo-dir: ${{ steps.local_repo.outputs.dir }}
```

Comment:
Why do you remove this part and duplicated the code?

It is a generic action to download full builds from repox.

Reply:
This was the root cause of the issue https://github.com/SonarSource/helm-chart-sonarqube/runs/6876350938?check_suite_focus=true

Sometimes, duplicating some code from elsewhere removes the complexity the dependency is introducing
And in this case, it is at the end 2 added lines of code:
![](https://files.slack.com/files-pri/T03LNT2JK-F03N2D2V74L/screenshot_2022-07-01_at_09.11.36.png)

I also believe that the download action is not a core feature of the release process (and the code relies on external tooling from Jfrog).